packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -91,7 +91,7 @@ class InteractiveViewer extends StatefulWidget {
     this.panEnabled = true,
     this.scaleEnabled = true,
     this.transformationController,
-    required Widget child,
+    required this.child, // ignore: tighten_type_of_initializing_formals
```

Comment:
Do I really need this ignore here or is there a right way to do this?  `child` is set only when using this constructor, otherwise it's null.  Vice versa for `builder`.

Reply:
My understanding is that you can avoid the ignore by writing the following (haven't tested it, though):

```suggestion
    required Widget this.child,
```